docs/typescript/statics-and-methods.md

```diff
@@ -80,3 +80,186 @@ const doc = new User({ name: 'test' });
 // Compiles correctly
 doc.updateName('foo');
 ```
+
+
+
+# Using `loadClass()` with TypeScript
+
+Mongoose supports applying ES6 classes to a schema using `schema.loadClass()`.
```

Comment:
A link to the `loadClass` function would likely be good here.
<https://mongoosejs.com/docs/api/schema.html#Schema.prototype.loadClass()>